elasticsearch/src/main/java/org/hibernate/search/elasticsearch/schema/impl/model/PropertyMapping.java

```diff
@@ -7,6 +7,8 @@
 package org.hibernate.search.elasticsearch.schema.impl.model;
 
 import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
```

Comment:
A few lines below (does not appear in the diff but just expand the lines), s/representic/representing/